docs/en/stack/security/authentication/overview.asciidoc

```diff
@@ -9,17 +9,23 @@ means (typically referred to as authentication tokens).
 The {stack} authenticates users by identifying the users behind the requests
 that hit the cluster and verifying that they are who they claim to be. The
 authentication process is handled by one or more authentication services called
-<<realms,_realms_>>. 
+<<realms,_realms_>>.
 
 You can use the native support for managing and authenticating users, or
 integrate with external user management systems such as LDAP and Active
-Directory. 
+Directory.
 
-{security} provides built-in realms such as `native`,`ldap`, `active_directory`, 
-`pki`, `file`, and `saml`. If none of the built-in realms meet your needs, you 
-can also build your own custom realm and plug it into the {stack}. 
+The {stack-security-features} provide built-in realms such as `native`,`ldap`,
+`active_directory`, `pki`, `file`, and `saml`. If none of the built-in realms
+meet your needs, you can also build your own custom realm and plug it in.
 
 When {security-features} are enabled, depending on the realms you've configured,
 you must attach your user credentials to the requests sent to {es}. For example,
-when using realms that support usernames and passwords you can simply attach 
+when using realms that support usernames and passwords you can simply attach
 {wikipedia}/Basic_access_authentication[basic auth] header to the requests.
+
+The {security-features} provide two services: the token service and the api key
+service. You can use these services to exchange the current authentication for a
+token or key, which is used for authentication without providing other means of
+authentication. These services are enabled by default when TLS/SSL is enabled
```

Comment:
> exchange the current authentication for a token... which is used for authentication without providing other means of authentication...

This sentence seems confusing to me.  It implies that you have been authenticated in order to get a token/key, so what other authentication would be required?

Reply:
Great, that's clearer.  Thanks!